<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>20</width>
      <height>861</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>1431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Simple Bound</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="simple_axis">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="simple_val">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>General</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="size">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="looper">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>#loops:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Create/Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="gavsLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Point:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>V0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Radial Bound</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cX">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cY">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>560</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Centre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="radiusBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>630</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>V0:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rV0_Box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>690</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>730</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>20</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tab1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>730</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tab2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>730</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="gavsBar">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>230</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="interact">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>270</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Interact</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioM">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Multigrid</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioG">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Gauss-Seidel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioJ">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Jacobi</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioS">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>SOR</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>-10</y>
       <width>171</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>radioM</zorder>
    <zorder>radioG</zorder>
    <zorder>radioJ</zorder>
    <zorder>radioS</zorder>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>X parallel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Y parallel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>560</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioFilled">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Filled</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioHollow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Hollow</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>10</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame_3</zorder>
    <zorder>radioFilled</zorder>
    <zorder>radioHollow</zorder>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>V0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioFilled_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Filled</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioHollow_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Hollow</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame_4</zorder>
    <zorder>radioFilled_2</zorder>
    <zorder>radioHollow_2</zorder>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rX">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="length">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rY">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Rectangular Bound</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recButoon">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>240</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>length:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recV0">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="width">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>height:</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>label</zorder>
   <zorder>simple_axis</zorder>
   <zorder>simple_val</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>size</zorder>
   <zorder>looper</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>gavsLabel</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>cX</zorder>
   <zorder>cY</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>radiusBox</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>rV0_Box</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>line_3</zorder>
   <zorder>tab1</zorder>
   <zorder>tab2</zorder>
   <zorder>gavsBar</zorder>
   <zorder>line_4</zorder>
   <zorder>interact</zorder>
   <zorder>groupBox</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_3</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>widget_4</zorder>
   <zorder>label_15</zorder>
   <zorder>label_16</zorder>
   <zorder>rX</zorder>
   <zorder>length</zorder>
   <zorder>rY</zorder>
   <zorder>label_17</zorder>
   <zorder>recButoon</zorder>
   <zorder>label_18</zorder>
   <zorder>recV0</zorder>
   <zorder>width</zorder>
   <zorder>label_19</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
